Dockerfile.ci

```diff
@@ -1045,6 +1045,9 @@ if [[ ${UPGRADE_BOTO=} == "true" ]]; then
     echo "${COLOR_BLUE}Upgrading boto3, botocore to latest version to run Amazon tests with them${COLOR_RESET}"
     echo
     pip uninstall --root-user-action ignore aiobotocore s3fs -y || true
+    # The limits for boto3 and botocore are set only because moto does not handle some changes implemented in botocore 1.31.1
```

Comment:
```suggestion
    # The limits for boto3 and botocore are set only because moto does not handle some changes implemented in botocore 1.32.1
```

Same for below

Reply:
Right ... Late in the night it was.